lib/resonate.ts

```diff
@@ -565,9 +577,16 @@ class ResonateContext implements Context {
 
       // trace
       const trace = this.startTrace(this.id);
-
+      // Check if test probability is passed in options
+      // only for testing purposes
+      const randomSeed = seedrandom();
```

Comment:
If we are going to use a library to generate seeded random numbers then we should pass this function in from the outside so that it can be a dev dependency. However, for the moment I recommend just using `Math.random()`.